qiita_ware/ebi.py

```diff
@@ -47,39 +48,7 @@ class NoXMLError(Exception):
 def clean_whitespace(s):
     """Standardizes whitespace so that there is only ever one space separating
```

Comment:
Since this isn't a private function would it be worthwhile to add the Parameters section in the numpydocs? (To clarify that `s` is a `str`)